dev-resources/template/1_db.yml

```diff
@@ -58,26 +61,42 @@ Parameters:
     AllowedValues:
       - true
       - false
-    Default: true
+    Default: false
 
 Conditions:
-  DBProvisioned: !Equals [!Ref DBEngineMode, "provisioned"]
-  DBSnapshotIdentifierProvided: !Not [!Equals [!Ref DBSnapshotIdentifier, ""]]
+  DBProvisioned:
+    !Equals [!Ref DBEngineMode, "provisioned"]
+  DBSnapshotIdentifierProvided:
+    !Not [!Equals [!Ref DBSnapshotIdentifier, ""]]
+  VPCIdProvided:
+    !Not [!Equals [!Ref VPCId, ""]]
+  DBSubnetsProvided:
+    !Not [!Equals [!Ref DBSubnets, !Ref "AWS::NoValue"]]
```

Comment:
This template fails remote validation on AWS (for some reason the cli validation doesn't catch it):
> Template error: every Fn::Equals object requires a list of 2 string parameters.

This is likely because you have something that acts like a comma-delimited list here (DBSubnets) but you are checking it like a string. Check what I do for the SetCORS condition over in the LRS template.

Also please attempt to run these to clear any other AWS-side validation errors.

Reply:
issue fixed via https://github.com/yetanalytics/lrsql/pull/276/commits/1a9d2f8f676130356fcddc57bd4b77b64c40f7f9 , also tested via lrsql-demo stack.